src/app/feed/feed.component.ts

```diff
@@ -243,6 +243,26 @@ export class FeedComponent implements OnInit, OnDestroy {
 		this.store.dispatch(new apiAction.UnSelectLightbox(event));
 	}
 
+	private filterresults(filtervalue) {
```

Comment:
@hemantjadon I have removed those. 
Previously, we were filtering out feeds with media from all the tweets. So, The number of feeds varied in the media tabs(sometimes 0,1 but not more than 5 most of the time) . Now, we are sending request to the server for tweets with specific media(images/videos) and it is returning those tweets only. This will increase the number of results.
 Moreover, previously, a problem with  pagination was going on. How would we implement pagination for media tabs?